refer/system-features.sh

```diff
@@ -0,0 +1,8 @@
+fn="features.txt"
+rm -rf $fn
+touch $fn
+for entry in $(ls *.json); do
+  n=$(echo ${entry%.*} | tr '[:lower:]' '[:upper:]')
+  f=$(grep '"' $entry | sed 's/ //g; s/"[,]*//g' | xargs)
+  echo "$n: $f" >> $fn
+done
```

Comment:
@siva19susi can you also check if this script produces the same result?

```bash
#!/bin/bash

# Define the name of the output file
fn="features.txt"

# Remove the output file if it already exists
if [ -f $fn ]; then
    rm $fn
fi

# Create an empty output file
touch $fn

# Loop over all the JSON files in the current directory
for entry in *.json; do
    # Extract the name of the JSON file without the extension, convert it to uppercase, and assign it to a variable
    n=$(echo ${entry%.*} | tr '[:lower:]' '[:upper:]')

    # Extract the values of all the fields in the JSON file, remove any spaces and quotes, and concatenate them into a single string separated by commas
    f=$(grep -o '[^[:space:]",]*' $entry | tr '\n' ',' | sed 's/,$//')

    # Write the name of the file and its fields to the output file
    echo "$n: $f" >> $fn
done
```

I asked chatgpt to make the script understandable and this is what it came up with :D

with the following clarification:

```
I added comments to explain what each part of the script does.
I added a check to see if the output file already exists before removing it.
I used the -o option with grep to extract all the field values from the JSON files, and I used tr to concatenate them into a single string separated by commas.
I removed the unnecessary spaces and quotes from the field values using a single sed command instead of two separate commands.
I used tr to remove the newline characters from the concatenated field string.
I added quotes around the $fn variable when writing to the output file to prevent issues with spaces in the filename.
```